python/test_dcr_template.py

```diff
@@ -0,0 +1,629 @@
+# LSST Data Management System
+# Copyright 2016 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+from __future__ import print_function, division, absolute_import
+import numpy as np
+from scipy import constants
+
+import lsst.afw.coord as afwCoord
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.meas.algorithms as measAlg
+from lsst.sims.photUtils import PhotometricParameters
+import unittest
+import lsst.utils.tests
+from .dcr_template import DcrModel
+from .dcr_template import DcrCorrection
+
+
+class _BasicBandpass:
+    """Dummy bandpass object for testing."""
```

Comment:
Yes, it looks like I can do that. It will change the values in the test data a little bit, but that probably makes it a better test.